packages/vizbuilder/package.json

```diff
@@ -17,25 +17,28 @@
     "release": "canon-release"
   },
   "dependencies": {
-    "@blueprintjs/core": "^1.39.0",
-    "@blueprintjs/labs": "^0.14.5",
+    "@blueprintjs/core": "^3.15.1",
+    "@blueprintjs/labs": "^0.16.3",
+    "@blueprintjs/select": "^3.8.1",
     "classnames": "^2.2.6",
     "d3-array": "^1.2.4",
     "d3plus-common": "^0.6.43",
     "d3plus-format": "^0.1.2",
     "d3plus-react": "^0.5.2",
     "fast-sort": "^1.5.3",
     "lodash": "^4.17.10",
+    "memoize-one": "^5.0.4",
     "mondrian-rest-client": "^1.1.0",
-    "pluralize": "^7.0.0",
+    "pluralize": "^8.0.0",
     "prop-types": "^15.6.0",
     "query-string": "^6.1.0",
-    "react-tiny-virtual-list": "^2.1.4",
+    "react": "^16.8.6",
```

Comment:
this shouldn't be added as a direct dependency? (it should be whichever version comes from canon)

Reply:
@frabarz can you speak to this? (especially after the convos on Slack)

Reply:
fixed on b43d576540dd00d5549870f6b8d8d104e28c25b9